notebooks/220-license-plate-recognition/220-license-plate-recognition.ipynb

```diff
@@ -0,0 +1,483 @@
+    "    print(f\"Origin image shape: {ori_img.shape}\")\n",
+    "\n",
+    "    # if invalid augmentation method passed in, the programme will stop.\n",
+    "    assert Augmentation in [\n",
+    "        \"None\",\n",
+    "        \"Laplace\",\n",
+    "        \"Log\",\n",
+    "        \"Gamma\",\n",
+    "        \"EquaHist\",\n",
+    "    ], \"Invalid Augmentation.\"\n",
+    "\n",
+    "    if Augmentation == \"None\":\n",
+    "        img = ori_img\n",
+    "\n",
+    "    elif Augmentation == \"Laplace\":\n",
+    "        kernel_sharpen = np.array([[0, 1, 0], [1, -4, 1], [0, 1, 0]])\n",
+    "        img = cv2.filter2D(ori_img, -1, kernel_sharpen)\n",
+    "\n",
+    "    elif Augmentation == \"Log\":\n",
+    "        img = 42 * np.log(1.0 + ori_img)\n",
+    "        img = np.uint8(img + 0.5)\n",
+    "\n",
+    "    elif Augmentation == \"Gamma\":\n",
+    "        lut = np.zeros(256, dtype=np.float32)\n",
+    "        for i in range(256):\n",
+    "            lut[i] = 0.00000005 * i**4.0\n",
+    "        img = cv2.LUT(ori_img, lut)\n",
+    "        img = np.uint8(img + 0.5)\n",
+    "\n",
+    "    elif Augmentation == \"EquaHist\":\n",
+    "        img0 = cv2.equalizeHist(ori_img[:, :, 0])\n",
+    "        img1 = cv2.equalizeHist(ori_img[:, :, 1])\n",
+    "        img2 = cv2.equalizeHist(ori_img[:, :, 2])\n",
+    "\n",
+    "        img = cv2.merge([img0, img1, img2])\n",
+    "\n",
+    "    # resize its shape to fit the model input requirements.\n",
+    "    resized_img = cv2.resize(img, (input_width, input_height))\n",
+    "\n",
+    "    # visualise how the image we process.\n",
+    "    ShowImage(ori_img, resized_img)\n",
+    "    trans_img = resized_img.transpose(2, 0, 1)\n",
+    "    input_img = np.expand_dims(trans_img, axis=0)\n",
+    "    return input_img"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Do the Inference\n",
+    "Match the output array into correct recognition results\n",
+    "Run license plate recognition model with the input and then match the result."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Process the Image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "input_img = ImageLoader(ImagePath=\"data/example.png\", Augmentation=\"None\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Input the Data and Get the Result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# an auxiliary blob that is needed for correct decoding.\n",
+    "# Set this to [0, 1, 1, ..., 1].\n",
+    "auxiliary_blob = np.array([0] + [1]*87)\n",
+    "# resize it to fit the shape: [88,1].\n",
+    "auxiliary_blob = np.resize(88,(88,1))\n",
```

Comment:
```suggestion
    "auxiliary_blob = np.resize(88, (88, 1))\n",
```
Add spaces after comas. Please also apply in comments and other places in notebook